src/main/java/com/park/easyrecruit/ejb/ApplicationBean.java

```diff
@@ -114,4 +113,21 @@ private Application getApplication(Integer positionId, String username) {
         return c;
     }
     
+    public void executeAction(String action, Integer positionId, String username) {
```

Comment:
This method signature should be further improved by restricting the inputs to what it is allowed - `String action` is too general. Either split it in two methods `approve(...)` and `reject(...)`, or use `status` instead of `String` as input: `updateStatus(Status status, ...)`.

Reply:
I just noticed this is will require changes all the way up to jsp files - too much for the last day of dev. Ignore the above comment. It is fine just as it is.